docs/go-logging-instrumentation.md

```diff
@@ -0,0 +1,68 @@
+---
+id: go-logging-instrumentation
+title: Scope Go Agent Logging Instrumentation
+sidebar_label: Logging Instrumentation
+---
+
+> Loggers and Standard Output/Error instrumentation is done global in the application. In order to differentiate the logs of a given test, only sequential tests are supported out of the box.
```

Comment:
I think we should add another section at the top of the file documenting how we automatically instrument `t.Log` (and friends), and that it is the recommended way to log as it both contains metadata about the source file and line number and is compatible with parallel tests.

Reply:
done